src/ib_module.F

```diff
@@ -1370,7 +1370,7 @@ subroutine ib_side_weno(stag,ix,jy,kz,c1,c2,rru,rrv,dumx,dumy,a,hadvorder,bndy,h
   ELSEIF( stag.eq.3 )THEN
 
     if( doit )then
-    !$acc parallel loop private(i,j,k,ubar,vbar)
+    !$acc parallel loop default(present) private(i,j,k,ubar,vbar)
```

Comment:
maybe add `gang vector` clause and `collapse` clause, and remove `private` clause?